src/components/manage/Tiles/HTML/Edit.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Edit html tile.
+ * @module components/manage/Tiles/HTML/Edit
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Editor from 'react-simple-code-editor';
+import { highlight, languages } from 'prismjs/components/prism-core';
+import 'prismjs/components/prism-markup';
+import { Button } from 'semantic-ui-react';
+
+import './styles.css';
```

Comment:
@ajayns Since we are doing all using the SemanticUI engine, we only insert HTML in the Pastanaga theme, that later might be overridable by integrators in Volto projects using `create-volto-app`.

Please place it in there (site.overrides is fine for now, we have to tidy up that file anyways)

Reply:
Added :+1